### ISSUE TITLE
Connection Issues?

### ISSUE THREAD

**Issue by Artie258** [vivinano/MudaeAutoBot#117]
![image](https://user-images.githubusercontent.com/92566031/137437003-8e32c728-e936-4488-864c-8594b75b8b2e.png)
Hello! I was running your code and I ran unto this issue, I updated discum but it did not work. I am not sure what might be the problem.

**Comment by vivinano** [comment]
That version specifically is the old version and not the new version please update to the new version

**Comment by Artie258** [comment]
Gotcha, thank you!

**Comment by Artie258** [comment]
Hello, sorry once again to bother you. I left my pc on overnight to see how was the bot doing and this is what it gave me 
![image](https://user-images.githubusercontent.com/92566031/137495657-c8725333-bab7-4aa6-b52f-f7255ab33abc.png)

**Comment by vivinano** [comment]
Yes it would be very appreciated :) specifically if it's an easy fix

**Comment by Artie258** [comment]
Hello! I fixed the ssl issue, I was told to install the following:

pip install ndg-httpsclient

pip install pyopenssl

pip install pyasn1

Now it does not give any error, but it has stayed like this since I fixed it, is it suppose to be doing the auto rolls?
![image](https://user-images.githubusercontent.com/92566031/137555328-3c3f7528-384c-4c11-a6ee-3f1e387fe471.png)

**Comment by vivinano** [comment]
Ah okay so it didn't install the dependencies alright cools

**Comment by Artie258** [comment]
Yes! I was told that there was a change in one of those so, reinstalling was recommended. The bot has been working very well. One question, does the bot do rolls as soon as the time hits xx:59 or is it configurable?

**Comment by Artie258** [comment]
Oh alright, thanks! I was kind of worried, as the bot sometimes did not want to roll. Thank you!

**Comment by Artie258** [comment]
Hello! I just wanted to ask, where in the code does the bot actually start roling? I left it overnight but I notcied that it only rolls once I open the bot, it just never rolls after that.
